Build Ocean 9.1.0 images

randomir opened the pull request:
In this PR, we:
- drop python 3.9, add python 3.14, default to 3.13
- drop debian bookworm/slim-bookworm (aka debian 12), default to trixie (aka debian 13) - a follow-up to #59
- install msvc build tools in windows images
- build ocean 9.1 images

randomir committed:
Generate Ocean 9.1.0 dockerfiles, tags and readme

randomir committed:
Drop Python 3.9 images, add Python 3.14 images, default to 3.13

randomir committed:
Drop debian bookworm and slim bookworm, default to trixie

randomir committed:
Install msvc build tools in windows images

thisac reviewed (approved):
Couldn't find anything missing or incorrect. LGTM!

randomir committed:
Reduce windows image size

Reduce from 16.5GB to 12.6GB (before compression). The base image
alone is 7.8GB.

randomir committed:
Optimize windows image size by removing dev components

SDK and build tools are not needed during runtime, just redist
libraries.

randomir committed:
Shrink windows images

Uncompressed size (of py314) image is now 8.6GB (that's 0.86GB on top
of the base windows image).
